src/index.html

```diff
@@ -1,15 +1,84 @@
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>HTML Form</title>
   <link rel="stylesheet" href="./style.css">
 </head>
+
 <body>
-  <h1>HTML Form</h1>
+  <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+    <fieldset class="form-section">
+      <legend>Personal information</legend>
+      <label> Surname:
+        <input class="form-question" name="surname" type="text" autocomplete="off" maxlength="40" minlength="2">
+      </label>
+      <label> Name:
+        <input class="form-question" name="username" type="text" autocomplete="off" maxlength="40" minlength="2">
+      </label>
+      <label> How old are You?
+        <input class="form-question" name="age" type="number" min="3" max="100" value="12">
+      </label>
+      <label> Full date of birth:
+        <input class="form-question" name="date" type="date">
+      </label>
+      <label> I accept the term of the agreement
+        <input class="form-question form-info" name="accept" type="checkbox" required>
+      </label>
+    </fieldset>
+    <fieldset class="form-section">
+      <legend>Registration</legend>
+      <label> E-mail:
+        <input class="form-question" name="mail" type="email" placeholder="email@example.com" required>
+      </label>
+      <label> Password:
+        <input class="form-question form-info" name="password" type="password">
+      </label>
+    </fieldset>
+    <fieldset class="form-section">
+      <legend>An interesting fact about you!</legend>
+      <label class="form-question"> Do you love cats?
+        <input id="love-cats" name="q-cats" type="radio">
+        <label for="love-cats">Yes</label>
+        <input id="hate-cats" name="q-cats" type="radio">
+        <label for="hate-cats">No</label>
+      </label>
+      <label class="form-question"> What is your favorite color?
+        <input name="q-color" type="color">
+      </label>
+      <label class="form-question"> What time do you go to bed?
+        <input name="bed-time" type="time">
+      </label>
+      <label class="form-question"> What are your favorite brand of cars?
+        <select name="cars" multiple>
+          <option value="BMW">BMW</option>
+          <option value="Audi">Audi</option>
+          <option value="Lada">Lada</option>
+        </select>
+      </label>
+      <label class="form-question form-info"> How do you rate our work?
+        <input name="rate-work" type="range" min="0" max="11" value="0">
+      </label>
+    </fieldset>
+    <fieldset class="form-section">
+      <legend>Additional info:</legend>
+      <label class="form-question">Comments:
+        <textarea name="comments"></textarea>
+      </label>
+      <label class="form-question form-info"> Would you recommend us?
```

Comment:
```suggestion
      <label> Would you recommend us?
```
form-question sets `margin-bottom: 10px`, form-info sets `margin-bottom: 10px`. you can remove both

Reply:
form-info set margin-bottom: 0px;